### PR TITLE
kernelci.org: Add documentation for the CIP instance

### PR DESCRIPTION
**Pull request by patersonc**
Add documentation for the CIP instance

### CODE REVIEW

**Comment by patersonc** [comment]
Hi @aliceinwire 

Do you have any other topics you'd like added?

Thanks, Chris

**Comment by aliceinwire** [comment]
@patersonc looks great for me!

**Review by aliceinwire — APPROVED** [review]

File: kernelci.org/content/en/docs/instances/cip.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+* *cip-gitlab*: This is a mirror of the upstream CIP tree hosted on GitLab. In
+addition this tree has extra, unique branches that are used to trigger specific
+CI build and test jobs
+
+## Configuration
+The build configurations (trees, branches, configurations etc.) are defined in
+[build-configs-cip.yaml](https://github.com/kernelci/kernelci-core/blob/main/config/core/build-configs-cip.yaml)
+from the [kernelci-core](https://github.com/kernelci/kernelci-core) project.
+
+The test configurations (devices, test cases, rootfs etc.) are defind in
```
Comment (aliceinwire) [inline kernelci.org/content/en/docs/instances/cip.md:37]: defind -> defined

**Review by aliceinwire — CHANGES_REQUESTED** [review]

File: kernelci.org/content/en/docs/instances/cip.md (hunk 1 above)
Comment (patersonc) [reply kernelci.org/content/en/docs/instances/cip.md]: oops! Fixed now

**Pull request merged**